test/include/dlaf_test/eigensolver/test_gen_eigensolver_correctness.h

```diff
@@ -37,11 +34,14 @@ namespace dlaf::test {
 template <class T, Device D, class... GridIfDistributed>
 void testGenEigensolverCorrectness(const blas::Uplo uplo, Matrix<const T, Device::CPU>& reference_a,
                                    Matrix<const T, Device::CPU>& reference_b,
-                                   dlaf::EigensolverResult<T, D>& ret, GridIfDistributed&... grid) {
+                                   dlaf::EigensolverResult<T, D>& ret, SizeType eval_index_begin,
+                                   SizeType eval_index_end, GridIfDistributed&... grid) {
   using dlaf::matrix::MatrixMirror;
   using dlaf::matrix::test::allGather;
   using dlaf::matrix::test::MatrixLocal;
 
+  DLAF_ASSERT(eval_index_begin == 0l, eval_index_begin);
+
```

Comment:
Is this asserted in teh algorithm itself as well? If yes, maybe we can leave it out here?

Reply:
Good point, I will remove it.